src/containers/treePage/index.tsx

```diff
@@ -326,6 +337,20 @@ const TreePage: React.FC<TreeProps> = ({
 
                             <SiteImageCarousel />
 
+                            <SelectorMapDisplay
+                              neighborhoods={neighborhoods}
+                              sites={sites}
+                              onMove={(pos: google.maps.LatLng) => {
+                                editSiteForm.setFieldsValue({
+                                  lat: round(pos.lat(), LAT_LNG_PRECISION),
+                                  lng: round(pos.lng(), LAT_LNG_PRECISION),
+                                });
+                              }}
+                              site={siteData.result}
+                              setMarker={setMapSearchMarker}
```

Comment:
⛏️ Similar to above - can we replace the values for `onMove` and `setMarker` with an empty lambda, something like `(_) => {}`? (you might need to add the eslint directive `// eslint-disable-next-line @typescript-eslint/no-empty-function` above both lines to pass the linter)